.github/workflows/merge.yml

```diff
@@ -143,7 +147,7 @@ jobs:
       - name: Determine which changes occurred
         if: ${{ inputs.modules_to_build == '' }}
         id: change_detection
-        uses: hv-actions/change-detection-builder@1.0.1
+        uses: hv-actions/change-detection-builder@1.0.0
```

Comment:
Probably not important but 1.0.1 is the latest one

Reply:
changed to 1.0.1